src/test/kotlin/com/amazon/opendistroforelasticsearch/notifications/TestUtils.kt

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.notifications
+
+import com.google.gson.JsonObject
+import com.google.gson.JsonParser
+import org.elasticsearch.client.Response
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStreamReader
+import java.nio.charset.StandardCharsets
+import java.util.Locale
+import kotlin.test.assertEquals
+
+class TestUtils {
```

Comment:
may be just use helper functions instead of class?

Reply:
Done